CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.3
+
+ANDI was not sent when aifa and asid were zeroed out
```

Comment:
Lets make it something vague like 
” Improve Android Identifiers collection”

Reply:
will change it on master